libraries/networking/src/udt/PacketList.cpp

```diff
@@ -13,17 +13,26 @@
 
 #include <QDebug>
 
-#include "Packet.h"
+#include "../NLPacket.h"
```

Comment:
We should probably move `PacketType` only to `NLPacketList` anyhow - doesn't necessarily have to be in this pull but that's not correct now that that has moved to `NLPacket`